src/main/resources/static/javascript/mainApp/admin.controller.js

```diff
@@ -257,8 +257,8 @@
                 return;
             }
 
-            groupingsService.invalidUhIdentifiers([sanitizedAdmin], (res) => {
-                if (!_.isEmpty(res)) {
+            groupingsService.getMemberAttributeResults([sanitizedAdmin], (res) => {
+                if (!_.isEmpty(res.invalid)) {
                     $scope.invalidInput = true;
                     return;
                 }
```

Comment:
Could you remove this invalid check and have $scope.displayAddModal handle it? Once that's done, I recommend removing checkInvalid from grouping.controller.js#$scope.addMembers().